global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "projects": [ "source" ],
```

Comment:
How did u find out about "projects" ? It is not documented https://docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json